patch-package
new file mode 100644
Binary files /dev/null and b/node_modules/grommet/.DS_Store differ
--- a/node_modules/grommet/components/Select/index.d.ts
+++ b/node_modules/grommet/components/Select/index.d.ts
@@ -8,11 +8,11 @@ export interface SelectProps {
   children?: (...args: any[]) => any;
   closeOnChange?: boolean;
   disabled?: boolean | number | string | object[];
-  disabledKey?: string | (...args: any[]) => any;
+  disabledKey?: string | ((...args: any[]) => any);
   dropAlign?: {top?: "top" | "bottom",bottom?: "top" | "bottom",right?: "left" | "right",left?: "left" | "right"};
   dropTarget?: object;
   focusIndicator?: boolean;
-  labelKey?: string | (...args: any[]) => any;
+  labelKey?: string | ((...args: any[]) => any);
   messages?: {multiple?: string};
   multiple?: boolean;
   onChange?: (...args: any[]) => any;
@@ -28,7 +28,7 @@ export interface SelectProps {
   dropHeight?: "xsmall" | "small" | "medium" | "large" | "xlarge" | string;
   value?: string | JSX.Element | object | string | object[];
   valueLabel?: React.ReactNode;
-  valueKey?: string | (...args: any[]) => any;
+  valueKey?: string | ((...args: any[]) => any);
   emptySearchMessage?: string;
 }
 
